### PR TITLE
Services: Continue separation of services into their own folders

### PR DESCRIPTION
**Pull request by purpasmart96**

### CODE REVIEW

File: src/common/logging/log.h [hunk 1]
```diff
@@ -57,13 +57,14 @@ enum class Class : ClassType {
...
+    Service_AM,                 ///< The AM (Application manager) service
     Service_PTM,                ///< The PTM (Power status & misc.) service
     Service_LDR,                ///< The LDR (3ds dll loader) service
     Service_NIM,                ///< The NIM (Network interface manager) service
-    Service_NWM,                ///< The NWM (Network manager) service
+    Service_NWM,                ///< The NWM (Network wifi manager) service
     Service_CFG,                ///< The CFG (Configuration) service
     Service_DSP,                ///< The DSP (DSP control) service
-    Service_HID,                ///< The HID (User input) service
+    Service_HID,                ///< The HID (Human input device) service
```
Comment (Subv) [inline src/common/logging/log.h]: Human Interface Device

File: src/common/logging/log.h [hunk 2]
```diff
@@ -57,13 +57,14 @@ enum class Class : ClassType {
     Service_APT,                ///< The APT (Applets) service
     Service_GSP,                ///< The GSP (GPU control) service
     Service_AC,                 ///< The AC (WiFi status) service
+    Service_AM,                 ///< The AM (Application manager) service
     Service_PTM,                ///< The PTM (Power status & misc.) service
     Service_LDR,                ///< The LDR (3ds dll loader) service
     Service_NIM,                ///< The NIM (Network interface manager) service
-    Service_NWM,                ///< The NWM (Network manager) service
+    Service_NWM,                ///< The NWM (Network wifi manager) service
```
Comment (Subv) [inline src/common/logging/log.h]: I think it's just NetWork Manager

Comment (purpasmart96) [inline src/common/logging/log.h]: Actually it stands for "WLAN" since it controls wlan.

**Comment by purpasmart96** [comment]
Done

**Comment by yuriks** [comment]
LGTM (Also builds cleanly rebased on master)

**Comment by bunnei** [comment]
LGTM

**Pull request merged**